### ISSUE TITLE
Change color of blast furnace conveyor belt click box when smithing gold bars without correct gloves equipped

### ISSUE THREAD

**Issue by CodyMace** [runelite/runelite#7815]
When smiting gold bars at the blast furnace with the blast furnace plugin the conveyor belt clickbox will be green even when goldsmith gauntlets are in inventory. 
Image: 
![current](https://user-images.githubusercontent.com/26229605/52604769-ddbdf600-2e31-11e9-88d5-a3c0fdaa6913.png)

I suggest that if goldsmith gauntlets are in the inventory of the player the clickbox should be red to warn the player they have the wrong gloves on. Once the gloves are equipped the clickbox should change to green. 

Image: 
![suggestion](https://user-images.githubusercontent.com/26229605/52604785-eb737b80-2e31-11e9-91fe-87c97cf97eca.png)

If the player doesn't have goldsmith gauntlets on or in their inventory the clickbox will just appear green like normal.

**Comment by CodyMace** [comment]
Possible fix:

https://github.com/runelite/runelite/pull/7816

**Comment by ypperlig** [comment]
I don't agree with this. The point of the furnace overlay is to show whether it's 'working' or not.
It would be better to show a new overlay indicating that gloves are missing imo.
